src/composables/useNotifications.ts

```diff
@@ -0,0 +1,23 @@
+import { reactive, toRefs } from 'vue';
+
+interface Notification {
+  message: string;
+  type: string;
+  timestamp: number;
+}
+
+const state = reactive({
+  items: [] as Notification[]
+});
+
+export function useNotifications() {
+  const notify = payload => {
```

Comment:
Could you make this a function like this:
```suggestion
  function notify(payload) {
```
This way it will be consistent with others functions